autogpt_platform/backend/backend/server/db_api.py

```diff
@@ -0,0 +1,64 @@
+from functools import wraps
+from typing import Any, Callable, Concatenate, Coroutine, ParamSpec, TypeVar, cast
+
+from backend.data import credit as C
+from backend.data import execution as E
+from backend.data import graph as G
+from backend.data.queue import RedisEventQueue
+from backend.util.service import AppService, expose
+from backend.util.settings import Config
+
+P = ParamSpec("P")
+R = TypeVar("R")
+
+
+class DatabaseAPI(AppService):
```

Comment:
This is the new service, it just wraps existing methods

Reply:
This is an outward facing API, it's just the port not expoesed. Any naming suggestion?